sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Microsoft.Azure.Management.RecoveryServices.Backup.csproj

```diff
@@ -6,13 +6,16 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.RecoveryServices.Backup</PackageId>
 		<Description>Provides developers with libraries for the updated recovery services backup feature under Azure Resource manager.</Description>
-		<Version>3.1.2-preview</Version>
+		<Version>3.1.3-preview</Version>
```

Comment:
I would suggest use 3.2.0 or even 4.0.0 for such a big change, especially you switched to a new api version

Reply:
Done